### PR TITLE
Low walls now have the correct pass flags

### LINKED ISSUE #111: Low walls block laser shots
I guess low walls shouldn't block any lasers but its especially an issue with setting up the sm as you need to crack open the chamber and emitter bay for the shots to hit

### PR DESCRIPTION
**Pull request by CapybaraExtravagante**
Used the passflags tables use, which sounds right to me.

closes #111

### CODE REVIEW

File: code/game/turfs/closed/wall/window_frames.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	smoothing_flags = SMOOTH_BITMASK|SMOOTH_OBJ
 	smoothing_groups = list(SMOOTH_GROUP_WINDOW_FRAMES)
 	canSmoothWith = list(SMOOTH_GROUP_WINDOW_FRAMES)
+	pass_flags_self = PASSTABLE | LETPASSTHROW
```
Comment (LemonInTheDark) [inline code/game/turfs/closed/wall/window_frames.dm:11]: what'd they have before?

Comment (CapybaraExtravagante) [reply code/game/turfs/closed/wall/window_frames.dm:11]: default which is null

Comment (LemonInTheDark) [reply code/game/turfs/closed/wall/window_frames.dm:11]: This might behave weirdly with dwarfs/monkys, you mind checking?

Comment (ArcaneMusic) [reply code/game/turfs/closed/wall/window_frames.dm:11]: Checked on local, dwarves and monkeys are able to walk over the low walls as they stand. Just the low walls work fine, 
image
Placing window glass on top by itself appears to cause visual artifacting to get left over on the mobile glass.
image
However, going through the construction steps appears to still behave fine. wall is solid and intact. No inconsistent throw behavior either.

In retrospect, the window artifacting is apparently just specific to windows... so the low walls looks good to go.

**Pull request merged**